setup/templates/steps/compose/01_config.html

```diff
@@ -0,0 +1,67 @@
+{% call macros.panel("info", "Step 2 - Initial configuration") %}
+<p>Before starting some variables must be set</p>
+
+<div class="form-group">
+  <label>Root path: </label>
+<!--   Validates path -->
+  <input class="form-control" type="text" name="root" value="/mailu" required pattern="^/[-_A-Za-z0-9]+(/[-_A-Za-z0-9]*)*">
+</div>
+
+<div class="form-group">
+  <label>Main mail domain</label>
+<!--   Validates domain name -->
+  <input class="form-control" type="text" name="domain" placeholder="e.g. mailu.io" 
+  		required pattern="^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$">
+</div>
+
+<div class="form-check form-check-inline">
```

Comment:
I do not think this belongs here. It is a very optional and quite experimental feature, far from the basic settings from this first section.